examples/collada.js

```diff
@@ -0,0 +1,89 @@
+/* global itowns, document, renderer, GuiTools, Promise */
+// # Simple Globe viewer
+
+// Define initial camera position
+// Coordinate can be found on https://www.geoportail.gouv.fr/carte
+// setting is "coordonnée geographiques en degres decimaux"
+
+// Position near Gerbier mountain.
+var positionOnGlobe = { longitude: 4.21655, latitude: 44.84415, altitude: 1500 };
+
+// `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
+var viewerDiv = document.getElementById('viewerDiv');
+
+// Instanciate iTowns GlobeView*
+var globeView = new itowns.GlobeView(viewerDiv, positionOnGlobe, { renderer: renderer });
+
+var menuGlobe = new GuiTools('menuDiv');
+
+var promiseElevation = [];
+
+menuGlobe.view = globeView;
+
+function addLayerCb(layer) {
+    return globeView.addLayer(layer).then(function addGui() {
+        var la = globeView.getLayers(function cl(l) { return l.id === layer.id; })[0];
+        if (la.type === 'color') {
+            menuGlobe.addImageryLayerGUI(la);
+        } else if (la.type === 'elevation') {
+            menuGlobe.addElevationLayerGUI(la);
+        }
+    });
+}
+// Add one imagery layer to the scene
+// This layer is defined in a json file but it could be defined as a plain js
+// object. See Layer* for more info.
+itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(addLayerCb);
+// Add two elevation layers.
+// These will deform iTowns globe geometry to represent terrain elevation.
+promiseElevation.push(itowns.Fetcher.json('./layers/JSONLayers/WORLD_DTM.json').then(addLayerCb));
+promiseElevation.push(itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addLayerCb));
+
+exports.view = globeView;
+exports.initialPosition = positionOnGlobe;
+
+function addMeshToScene() {
+    var model;
+    // loading manager
+    var loadingManager = new itowns.THREE.LoadingManager(function _addModel() {
+        globeView.scene.add(model);
+        globeView.notifyChange(true);
+    });
+    // collada
+    var loader = new itowns.THREE.ColladaLoader(loadingManager);
+
+    // get the position on the globe, from the camera
+    var cameraTargetPosition = globeView.controls.getCameraTargetGeoPosition();
+
+    // position of the mesh
+    var meshCoord = cameraTargetPosition;
+    meshCoord.setAltitude(cameraTargetPosition.altitude());
```

Comment:
Unfortunately this doesn't work in the general case. Especially, if you throttle the network with chrome devtools, you'll see that this gets resolves before the correct elevation texture arrives, causing the house to be under ground. 

To fix it, you need to implement a child layer of the right elevation layer (or of the base layer) to be able to react to any elevation change at this point.

Other problem: if the user move the camera before the house is loaded, it will get loaded at the wrong position.

That makes me think we need an easier mechanism for laying objects on grounds at a specific position, that will basically doing all the boilerplate of adding the mesh to the right tile, put it in local coordinates and update its z when / if this tile get some elevations.

For this PR, I'm not requiring this. This is a problem we already have in other examples anyway.

Reply:
Actually I also reproduce with firefox, everytime...

Reply:
yes indeed, we know it's a problem. I can remove all the layers in the example to reach the goal of this PR to load a collada.

Reply:
Or you can do this:

> To fix it, you need to implement a child layer of the right elevation layer (or of the base layer) to be able to react to any elevation change at this point.

This works, for example, added at the end of `addMeshToScene`:

```js
window.position = new itowns.Coordinates('EPSG:4326',  positionOnGlobe.longitude, positionOnGlobe.latitude);
globeView.addLayer({
        update: (context, layer, element) => {
            if (element.extent.isPointInside(position) && element.visible && element.material &&  element.material.visible) {
                itowns.DEMUtils.placeObjectOnGround(globeView.wgs84TileLayer, globeView.referenceCrs, model, {}, [element]);
            }
        }
    },
    globeView.wgs84TileLayer);
```
The beauty of it: this way you don't need to wait for elevationPromise to load your collada. 

There is still the issue of the camera: it's not placed correctly if the elevation layer arrives too slowly (I reproduce it in Firefox and in chrome with network slowed)... Maybe we can update its position the first time we have an element with an elevation texture in the update above ?